view/adminhtml/templates/system/config/launch_yotpo_button.phtml

```diff
@@ -6,6 +6,7 @@
 <?php if (!$block->isStoreScope()) : ?>
     <p class="note"><?= __('Yotpo can only be connected, enabled, or disabled via the Store View menu.<br>To connect your App Key and API secret, first select the relevant Store View.') ?></p>
     <p></p>
+    <!--style>tr#row_yotpo_settings_active,tr#row_yotpo_settings_app_key,tr#row_yotpo_settings_secret{display:none!important;}</style-->
```

Comment:
do not add unneeded comments.

Reply:
It's was a mistake on the commit, I pushed a fix few minutes after that...